boards/arm/stm32f3_disco/stm32f3_disco.dts

```diff
@@ -187,3 +187,8 @@
 	pinctrl-0 = <&adc1_in1_pa0>;
 	status = "okay";
 };
+
+&dac1 {
+	status = "okay";
+	pinctrl-0 = <&dac_out1_pa4>;
```

Comment:
PA4 will probably conflict with 
&spi1 {
	pinctrl-0 = <&spi1_nss_pa4 &spi1_sck_pa5
		     &spi1_miso_pa6 &spi1_mosi_pa7>;
	status = "okay";
};

Reply:
@ikmdani Is there another pin that could be available to avoid this conflct?